src/reducers/index.js

```diff
@@ -209,6 +212,22 @@ const slice = createSlice({
       state.scrolling = true;
       state.scrollX = scrollX;
     },
+    updateRowHeights(state, action) {
+      const firstUpdatedRowIndex = action.payload || 0;
+      if (firstUpdatedRowIndex >= state.getInternal().rowUntilOffsetsAreExact) {
+        return;
+      }
+      // Invalidate all the previous computed row heights till the updated row
+      state.getInternal().rowUntilOffsetsAreExact = firstUpdatedRowIndex;
+      // Refresh the current scroll position according to the new row heights
+      const currentScrollY =
+        state
+          .getInternal()
+          .rowOffsetIntervalTree.sumUntil(state.firstRowIndex) -
+        state.firstRowOffset;
+      const scrollAnchor = scrollTo(state, currentScrollY);
```

Comment:
shouldn't this pass the third param as `props.useExactRowHeight` or `true` ?

Reply:
Yes. You are right. I have changed this. Thank you